src/components/Footer/index.js

```diff
@@ -1,34 +1,42 @@
 import React from "react";
-import { Row, Col, Divider } from "antd";
 import { Link } from "react-router-dom";
 import BrandName from "../../helpers/brandName";
-import {
-  GithubOutlined,
-  UnorderedListOutlined,
-  BugOutlined,
-  PhoneOutlined,
-  MailOutlined,
-  HomeOutlined,
-  CheckOutlined,
-  LockOutlined,
-  QuestionOutlined,
-  CopyrightCircleOutlined,
-} from "@ant-design/icons";
+import Grid from "@material-ui/core/Grid";
+import Divider from "@material-ui/core/Divider";
+import { makeStyles } from "@material-ui/core/styles";
+import GitHubIcon from "@material-ui/icons/GitHub";
+import BugReportOutlinedIcon from "@material-ui/icons/BugReportOutlined";
+import ListOutlinedIcon from "@material-ui/icons/ListOutlined";
+import HelpOutlineOutlinedIcon from "@material-ui/icons/HelpOutlineOutlined";
+import CheckOutlinedIcon from "@material-ui/icons/CheckOutlined";
+import LockOutlinedIcon from "@material-ui/icons/LockOutlined";
+import PhoneEnabledOutlinedIcon from "@material-ui/icons/PhoneEnabledOutlined";
+import MailOutlineOutlinedIcon from "@material-ui/icons/MailOutlineOutlined";
+import HomeOutlinedIcon from "@material-ui/icons/HomeOutlined";
+import CopyrightOutlinedIcon from "@material-ui/icons/CopyrightOutlined";
```

Comment:
Is there any specific reason to have imports like this way?

Reply:
This is discussed in this thread. https://github.com/scorelab/Codelabz/pull/75#discussion_r638543643
It is done since the Material-UI docs says we should use it to decrease bundle size.